### PR TITLE
[3.1] Category Image Path: does not work #2726

### PR DESCRIPTION
**Pull request by xillibit**

### CODE REVIEW

**Comment by xillibit** [comment]
@GJSchaller : can-you test it please ?

**Comment by GJSchaller** [comment]
Testing - using Crypsis - set the path to "klarington" in the Crypsis config.

Path to Category Icons should be:

(joomla root)/media/kunena/klarington

but instead it is:

(joomla root)/components/com_kunena/template/crypsis/media/topic_icons/default/system/

For Blue Eagle, it is:

(joomla root)/media/kunena/topic_icons/default/system/normal.png

I may have goofed the files - if you want access to my test site, please let me know.

**Comment by 810** [comment]
i want to move all crypsis media to media/kunena

Then there is no issue any, also i have a new pr, that's improving the topic icons

**Comment by xillibit** [comment]
I have updated it, for blue eagle and old templates this won't change nothing and till use setting under Kunena configuration panel but for Crypsis and new templates this path need to be set under template settings.

File: components/com_kunena/site/layout/category/index.php [hunk 1]
```diff
@@ -43,9 +43,14 @@ public function getPaginationObject($maxpages)
...
-		$this->ktemplate = KunenaFactory::getTemplate();
-		$defaultcategoryicon = $this->ktemplate->params->get('DefaultCategoryicon');
-		if ($category->getNewCount())
+		$template = KunenaFactory::getTemplate();
+		$catimagepath = $template->params->get('catimagepath');
+
+		$path = JPATH_ROOT . '/media/kunena/' . $catimagepath . '/';
+		$uri  = JUri::root(true) . '/media/kunena/' . $catimagepath . '/';
+
+		if (!$thumb)
```
Comment (810) [inline components/com_kunena/site/layout/category/index.php]: should be `if ($category->getNewCount())` I think

Comment (xillibit) [inline components/com_kunena/site/layout/category/index.php]: Instead of `php if (!thumb)` ?

Comment (810) [inline components/com_kunena/site/layout/category/index.php]: yeah, because i can't see what $thumb is. before it was also with that code.

**Comment by xillibit** [comment]
@810 : should be ok now

**Comment by 810** [comment]
can't see the option on edit template

**Comment by xillibit** [comment]
The option COM_KUNENA_CATIMAGEPATH_LEGACY is on Kunena configuration

**Comment by rich20** [comment]
- [ ] Failed - Crypsis shows this error instead of category icons

Notice: Undefined variable: defaultcategoryicon in components/com_kunena/layout/category/index.php on line 71

**Comment by xillibit** [comment]
Try now

**Comment by rich20** [comment]
- [x] Crypsis shows error - Succes
- [x] Image path - Success

**Pull request merged**